components/prize_pool/commons/prize_pool.lua

```diff
@@ -705,6 +765,62 @@ function Placement:_parseOpponentArgs(input, date)
 	return Opponent.resolve(opponentData, date)
 end
 
+function Placement:_getLpdbData()
+	local entries = {}
+	for _, opponent in ipairs(self.opponents) do
+		local participant, image, imageDark, players
+		local playerCount = 0
+
+		if opponent.opponentData.type == Opponent.team then
```

Comment:
until line 791: move into opponent module to be able to adjust according to wiki needs (e.g. faction handling, further opponent types)
—> leave comment?

Reply:
Well this code is for the soon to be deprecated "participant" (in favor of opponent). Module Opponent would deal the proper opponent fields. The LPDB injector can set any extradata fields (such as faction) or rewriting the default handling.